sarpy/utils/nitf_utils.py

```diff
@@ -20,7 +21,7 @@
 
 
 # Custom print function
-print_func = print
+print_func = print_function
```

Comment:
Print is a function in Python 2.7, and many things in sarpy require a fairly modern Python 2.7 version. I will actively avoid partial Python 2 issues, particularly since it's beyond its end of life.

This was there as some kind of left-over in Austin's code, and didn't seem important enough to discuss with him. It seemed like he wanted to ability to Monkey patch for custom print behavior, so I left this in there.

Reply:
`print` is a keyword in python 2.7 but a function in python 3.  You can import from `__future__` to get print as a function.

Reply:
Hi Mark, thanks for your input. As I stated above, print was incorporated as a function in Python 2.7.(some number). This was part of a whole host of changes as part of the python 2 -> 3 transition preparation, many of which were breaking changes right in the middle of Python 2.7 versioning.

It's true that prior to some point in Python 2.7, print was merely a statement. Unfortunately, there are definitely other things in sarpy that won't work before this stage of Python 2.7. Compliance of these things is a total pain, and ultimate not worth it at all because Python 2 is already well beyond its end of life.

Reply:
For what it's worth, this bit us again in another file https://github.com/ngageoint/sarpy/blob/0edefee386834c682fe03c295517f4479407c027/sarpy/utils/review_class.py#L13
The documentation https://docs.python.org/2.7/reference/simple_stmts.html#the-print-statement says it's still a statement at the end of 2.7.

When I try it, I get this:
```
Python 2.7.18 |Anaconda, Inc.| (default, Apr 23 2020, 22:42:48) 
[GCC 7.3.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> print "HI"
HI
>>> print_func = print
  File "<stdin>", line 1
    print_func = print
                     ^
SyntaxError: invalid syntax
```

But if I import it from `__future__`
```
Python 2.7.18 |Anaconda, Inc.| (default, Apr 23 2020, 22:42:48) 
[GCC 7.3.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from __future__ import print_function
>>> print_func = print
>>> type(print)
<type 'builtin_function_or_method'>
```
Python 2.7.18 is the latest at this time and was released in April of 2020.

How are you able to set up a python 2.7 environment that has a print function by default?